packages/editor-ui/src/constants.workflows.ts

```diff
@@ -0,0 +1,208 @@
+import { NodeConnectionType } from 'n8n-workflow';
+import type { INodeUi, IWorkflowDataCreate } from './Interface';
+
+export const EASY_AI_WORKFLOW_JSON: IWorkflowDataCreate = {
+	name: 'Demo: My first AI Agent in n8n',
+	meta: {
+		templateId: 'PT1i+zU92Ii5O2XCObkhfHJR5h9rNJTpiCIkYJk9jHU=',
+	},
+	nodes: [
+		{
+			parameters: {
+				operation: 'getAll',
+				calendar: {
+					__rl: true,
+					mode: 'list',
+				},
+				returnAll: true,
+				options: {
+					timeMin:
+						"={{ $fromAI('after', 'The earliest datetime we want to look for events for') }}",
+					timeMax: "={{ $fromAI('before', 'The latest datetime we want to look for events for') }}",
+					query:
+						"={{ $fromAI('query', 'The search query to look for in the calendar. Leave empty if no search query is needed') }}",
+					singleEvents: true,
+				},
+			},
+			type: 'n8n-nodes-base.googleCalendarTool',
+			typeVersion: 1.2,
+			position: [880, 220],
+			id: '0d7e4666-bc0e-489a-9e8f-a5ef191f4954',
+			name: 'Google Calendar',
+		},
+		{
+			parameters: {
+				options: {},
+			},
+			id: '5b410409-5b0b-47bd-b413-5b9b1000a063',
+			name: 'When chat message received',
+			type: '@n8n/n8n-nodes-langchain.chatTrigger',
+			typeVersion: 1.1,
+			position: [360, 20],
+			webhookId: 'a889d2ae-2159-402f-b326-5f61e90f602e',
```

Comment:
Well, even if it is, it's fine I guess. I was worried about cloud but even there this will only be there once per instance

Reply:
True, and I think these will be ignored either way. Let me check that and remove to clean it up as much as possible.

Reply:
Just checked and the id's will be kept when importing so keeping this to avoid additional id assignment. Also to clarify, these are just workflow-specific, not instance